packages/themed-components/src/CollapsiblePanel/CollapsiblePanel.tsx

```diff
@@ -145,6 +148,7 @@ const HeaderContainer: React.FC<{
 	expanded?: boolean
 	onClick: () => void
 	onKeyDown: (ev: React.KeyboardEvent) => void
+	styles?: React.CSSProperties
```

Comment:
If the style is just a single React.CSSProperties object, I would name it `style` to map directly to the style DOM attribute. `styles` is more appropriate with the nested style objects that Fluent uses